### ISSUE TITLE
Changing range of dates

### ISSUE THREAD

**Issue by hancelpv** [gregrahn/tpcds-kit#46]
With the default setting, data is generated from 1998-01-02 to 2003-01-02.
Is it possible to generate data for a different range of dates ?
for instance 2015-01-01 to 2020-01-01..

**Comment by gregrahn** [comment]
You can:

1) load and do date math with SQL either in-line or via staging tables
2) change the range in [constants.h](https://github.com/gregrahn/tpcds-kit/blob/master/tools/constants.h#L41-L42) and recompile

1 is sure to work, you will have to test 2.
Do note this will likely make the queries return 0 rows as they are aligned to the default date window.

**Comment by hancelpv** [comment]
@gregrahn  Thank you so much for your prompt reply !
I used method 2, changed the sysdate to a future date and obtained data for future dates as well.
I have one more query regarding promotion data.

My date range is 2015-01-01 to 2019-04-16 in constants.h.
When I generated data at scale 1, promotion.dat file has the latest record 27-03-2015.
Is there anyway we can generate promotion records too for the future/current dates ?
